int32 lPosiX 
int32 lPosiY
int16 sTheta
int16 sRealLinearVel
int16 sRealAngularVel
int16 sVoltIn
int16 sLeftMotCur
int16 sRightMotCur
uint8 byUS1
uint8 byUS2
uint8 byUS3
uint8 byUS4
uint8 byPlatStatus
uint8 byDocStatus
uint8 byLeftMotStatus
uint8 byRightMotStatus
int32 lLeftMotPosi
int32 lRightMotPosi
uint8 byLeftIOMonitor
uint8 byRightIOMonitor
